_layouts/home.html

```diff
@@ -1,78 +1,93 @@
 ---
 layout: default
 ---
+<div id="home">
+{% include header.html %}
 
+<div class="header-image">
+  <div class="header-image-caption">
+    art made with mypaint by <a href="http://nrekkvan.deviantart.com">nrekkvan</a>
+  </div>
+</div>
 
-<div class="home pure-g">
-  <div class="l-box">
+<div id="content" class="content">
+  {% include title.html %}
 
-    <div class="posts description pure-u-1">
-      {{ site.description }}
-    </div>
+  <div class="home pure-g">
+    <div class="l-box">
 
-    <div class="posts pure-u-1">
-      <h2>Features</h2>
-      <ul>
-        {% for item in page.features %}
-        <li>{{ item }}</li>
-        {% endfor %}
-      </ul>
-    </div>
-
-    <div class="posts pure-u-1">
+      <div class="posts description pure-u-1">
+        <p class="lead">
+          <em>MyPaint is a nimble, distraction-free, and easy tool for digital painters.</em> It supports graphics tablets made by Wacom, and many similar devices. Its brush engine is versatile and configurable, and it provides useful, productive tools.
```

Comment:
You can use the `markdownify` Jekyll liquid filter that way you can apply markdown to the _config.yml file.

```
Markdownify: Convert a Markdown-formatted string into HTML.

Input:
{{ "Hello **Jekyll**" | markdownify }}

Output:
Hello <strong>Jekyll</strong>
```
Cheat Sheet: http://jekyll.tips/jekyll-cheat-sheet/